crates/driver/src/infra/solver/dto/auction.rs

```diff
@@ -352,6 +354,7 @@ struct Order {
     full_sell_amount: eth::U256,
     #[serde(skip_serializing_if = "Option::is_none")]
     fee_policies: Option<Vec<FeePolicy>>,
+    created: Option<u32>,
```

Comment:
Why do these fields need to be sent down to the solver engines as well? The sorting logic will happen in the driver, so the information doesn't need to flow downstream.